Resources/Prototypes/SS220/Entities/Objects/Tools/rcd-ce.yml

```
@@ -0,0 +1,26 @@
+# © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+- type: entity
+  id: RCDCE
+  parent: RCD
+  name: Advanced RCD
+  description: A Senior Engineer's bluespace-enhanced rapid construction device that passively generates its own compressed matter.
+  suffix: AutoRecharge
+  components:
+  - type: Sprite
+    sprite: SS220/Objects/Tools/RCDCE.rsi
+  - type: Clothing
+    sprite: SS220/Objects/Tools/RCDCE.rsi
+    quickEquip: false
+    slots:
+    - Belt
+  - type: LimitedCharges
+    maxCharges: 30
+    charges: 30
```

```suggestion
    maxCharges: 20
    charges: 20
```
Это изначальные статы такого РСУ, что у боргов было